openhantek/src/glscope.cpp

```diff
@@ -40,15 +40,15 @@ void GlScope::fixOpenGLversion(QSurfaceFormat::RenderableType t) {
     // Prefer full desktop OpenGL without fixed pipeline
     QSurfaceFormat format;
     format.setSamples(4); // Antia-Aliasing, Multisampling
-    format.setProfile(QSurfaceFormat::CoreProfile);
     if (t==QSurfaceFormat::OpenGLES) {
-        format.setVersion(2, 0);
```

Comment:
The version restrictions don't mean the exact version but a minimum required version. And it will not work with OpenGL es 1.0

Reply:
Actually OpenGL ES 1.0 doesn't work in the way how it's written in the code

``` c++
   const char *vshaderES = R"(
           #version 100
            attribute highp vec3 vertex;
            uniform mat4 matrix;
            void main()
            }
      )";
```

It will tell you about compilation error you need to define precision in vertex (something like that)
when you define precision it throws to another error

that's why I've changed version to 1.2
```
 #version 120
```
I agree it's not good solution to increase version, but I didn't find solution
Sure maybe you know because my experience in OpenGL just one day :)